config/systems-uenv/hohgant-uenv.py

```diff
@@ -25,7 +24,7 @@
 image_name = image_path.stem
 
 # Options for the Slurm plugin to mount the Squashfs uenv image
-uenv_access = [f'--uenv-file={uenv_file}', f'--uenv-mount={uenv_mount}']
```

Comment:
This will not be needed once https://github.com/eth-cscs/cscs-reframe-tests/pull/124 is merged.

Reply:
merged